cmd/endorse.go

```diff
@@ -77,8 +77,8 @@ func (f *endorseCommand) AddFlags(cmd *cobra.Command) {
 	cmd.PersistentFlags().AddGoFlag(
 		amdProductVar(&ec.SevSnp.Product, "snp_product", sgpb.SevProduct_SEV_PRODUCT_MILAN,
 			"SEV-SNP product line. One of Milan, Genoa"))
-	cmd.PersistentFlags().BoolVar(&ec.Tdx.IncludeEarlyAccept, "tdx_include_early_accept", false, "If true, adds MRTD values for all supported machine shapes with all memory configured for acceptance in the firmware.")
-	cmd.PersistentFlags().StringSliceVar(&ec.Tdx.MachineShapes, "tdx_machine_shapes", nil, "GCE machine shapes whose measurement-relevant configuration should be enumerated for endorsement.")
+	cmd.PersistentFlags().BoolVar(&ec.Tdx.IncludeEarlyAccept, "tdx_include_early_accept", false, "DEPRECATED: If true, adds MRTD values for all supported machine shapes with all memory configured for acceptance in the firmware.")
+	cmd.PersistentFlags().StringSliceVar(&ec.Tdx.MachineShapes, "tdx_machine_shapes", nil, "DEPRECATED: GCE machine shapes whose measurement-relevant configuration should be enumerated for endorsement.")
```

Comment:
The "DEPRECATED" message is pretty confusing. What should a user do if they see that the flag is deprecated? Just not use it? Use a different flag?

Reply:
Updated the text for the age of VMM needed for its relevance.